src/runtime/runtime_stm32f103xx.go

```diff
@@ -44,13 +44,145 @@ func initCLK() {
 	}
 }
 
+const tickMicros = 1000
+
+var (
+	timestamp        timeUnit // nanoseconds since boottime
+	timerLastCounter uint32   // 24 bits ticks
```

Comment:
1. "nanoseconds since boottime" seems wrong, timeUnit should be given in `tickMicros` per nanosecond. If the value is correct, `timestamp` would be given in microseconds since boottime.
2. `timerLastCounter` is not a 24-bit timer? It's used for the RTC, so I guess it must be 32-bit.